### DISCUSSION TITLE
Slideshow starts paused

### DISCUSSION THREAD

**Discussion by s1nbad** [d2phap/ImageGlass#1185] [Q&A] [answered]
When I open IG with   key="IsSlideshow" value="True"   it opens in slideshow but is paused.  Is there anyway I can change this behaviour ?

**Comment by d2phap** [comment, marked as answer]
ImageGlass does not support startup in Slideshow mode. You can follow this #740 for future update.
Thanks!

**Reply by s1nbad** [reply]
Understood, shame, although not supported (apart from being paused) it seems to work well in this configuration.  I am able to start in Slideshow mode and in a Window (as apposed to Fullscreen).  I notice other users discussing workarounds for both these scenarios, hopefully you will take a closer look at some time.  Starting in Slideshow mode and in a windowed environment would be welcome enhancements.  Great project.
